src/reporter/reporters/WarningsReporter.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseReporter } from "@src/reporter/reporters/BaseReporter";
```

Comment:
This will break the plugin.

```suggestion
import { BaseReporter } from "../reporters/BaseReporter";
```